src/css/style.scss

```diff
@@ -0,0 +1,26 @@
+html, body {
```

Comment:
You should give the next, prev elements class names (with some prefix like "crc-" to make sure it's unique when pull into other projects) and reference those class names in this scss file.

Also wrap the carousel itself with a div elements that's position:relative to prepare for when the carousel is used outside of iframes (on other projects)